devito/ops/compiler.py

```diff
@@ -0,0 +1,17 @@
+from devito.parameters import configuration
+
+
+__all__ = ['CompilerOPS']
+
+
+class CompilerOPS(configuration['compiler'].__class__):
+    def __init__(self, *args, **kwargs):
+        super(CompilerOPS, self).__init__(*args, **kwargs)
+
+    def jit_compile(self, soname, code, hcode):
+        super(CompilerOPS, self).jit_compile(soname, code)
+        target = str(self.get_jit_dir().joinpath(soname))
+        h_file = "%s.h" % (target)
+        kernel_file = open(h_file, "w")
+        for kernel in hcode:
+            kernel_file.write(kernel)
```

Comment:
you're not compiling anything now?

Reply:
Now i just write the  h. file after the creation of the .c file in the jit-compile function. Tried to keep them near each other. Further on i plan to add the call to the ops translator and so on on this compiler, if that makes sense.